go.mod

```diff
@@ -236,7 +236,7 @@ replace (
 	github.com/kr/pretty => github.com/kr/pretty v0.1.0
 	github.com/kr/pty => github.com/kr/pty v1.1.5
 	github.com/kr/text => github.com/kr/text v0.1.0
-	github.com/kubernetes-csi/csi-proxy/client => github.com/kubernetes-csi/csi-proxy/client v0.0.0-20200330215040-9eff16441b2a
+	github.com/kubernetes-csi/csi-proxy/client => github.com/kubernetes-csi/csi-proxy/client v0.0.0-20200907051941-1c32ab57cb4f
```

Comment:
this replace is needed?

Reply:
yes, while in this PR, there is no vendor code change since smb client is still alpha and remains the same.